app/server.js

```diff
@@ -21,7 +21,7 @@ pool.connect().then(function () {
   console.error("Database connection error:", error);
 });
 
-app.use(express.static("public"));
```

Comment:
Can you add this line back please? I need this for the code I'm working on for the bidding logic